### PR TITLE
[EDU-3163] feat: automate secure sdk pt

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue: 

- [EDU-3163](https://github.com/aziontech/docs/pull/482)

Changes

- Traduz docs de automate secure - edge functions e APIs

[EDU-3163]: https://aziontech.atlassian.net/browse/EDU-3163?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/secure-jornada/automate/sdk.mdx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+permalink: /documentacao/produtos/secure/automate/sdk/
+---
+
+import Button from '~/components/Button.astro'
+
+<Button href="/pt-br/documentacao/devtools/sdk/go/" text="ir para a documentação de referência do Azion SDK" variant="secondary"></Button>
+
+O Azion SDK para Go serve como um poderoso conjunto de ferramentas para desenvolvedores, projetado para simplificar a integração dos serviços de API da Azion em aplicações da linguagem de programação Go.
+
+Esta ferramenta facilita a interação com as APIs da Azion, capacitando os desenvolvedores a gerenciar eficientemente os recursos de segurança por meio de aplicações programáveis.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/secure-jornada/automate/sdk.mdx:16]:
```suggestion
O Azion SDK para Go serve como um poderoso conjunto de ferramentas para desenvolvedores, projetado para simplificar a integração dos serviços de API da Azion em aplicações da linguagem de programação Go.

Esta ferramenta facilita a interação com as APIs da Azion, capacitando os desenvolvedores a gerenciar eficientemente os recursos de segurança por meio de aplicações programáveis.

<Button href="/pt-br/documentacao/devtools/sdk/go/" text="saiba mais sobre o Azion SDK" variant="secondary"></Button>
```

**Review by MarianaAguilera — APPROVED** [review]
deixei um par de sugestões.

File: src/content/docs/pt-br/pages/secure-jornada/automate/sdk.mdx [hunk 2]
```diff
@@ -0,0 +1,154 @@
+---
+title: Automatize a segurança através do Azion SDK
+description: O Azion SDK para a linguagem Go facilita a interação com as APIs da Azion, capacitando os desenvolvedores a gerenciar eficientemente os recursos de segurança por meio de aplicações programáveis.
+meta_tags: edge, segurança, sdk
+namespace: documentation_secure_automate_sdk
+menu_namespace: secureMenu
+permalink: /documentacao/produtos/secure/automate/sdk/
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/secure-jornada/automate/sdk.mdx:7]:
```suggestion
permalink: /documentacao/produtos/secure/automatize/sdk/
```

**Review by bru-andrade — APPROVED** [review]
👯‍♀️

**Pull request merged**